PowerPoint_Adapter/AdapterActions/Push.cs

```diff
@@ -119,34 +117,25 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
                 try
                 {
                     if (m_OutputFileSettings != null)
-                        presentationDoc.Clone(m_OutputFileSettings.GetFullFileName());
+                        presentationDoc.Clone(m_OutputFileSettings.GetFullFileName()).Dispose();
                     else if (m_OutputStream != null)
                     {
                         presentationDoc.Clone(m_OutputStream);
                         m_OutputStream.Position = 0;
                     }
-
                 }
-                catch (Exception e)
+                catch (Exception ex)
                 {
-                    BH.Engine.Base.Compute.RecordError("Could not save the changes: " + e.Message);
+                    BH.Engine.Base.Compute.RecordError(ex, "An error occurred when trying to save the presentation.");
```

Comment:
Can we keep the exception's message in the error message? 
This can help understand where the error is coming from when in the UI.